### PR TITLE
feat(cfp): adjust cfp with new design

### LINKED ISSUE #24: Call for proposal with new design

### PR DESCRIPTION
**Pull request by jeeehaan**
Closes #24

### CODE REVIEW

File: app/routes/cfp._index.tsx [hunk 1]
```diff
@@ -19,19 +19,34 @@ export const meta: V2_MetaFunction = () => {
...
+            </p>
+            <Button
+              asChild
+              size="lg"
+              className="mt-10 w-full md:mt-28 md:w-auto lg:mt-8"
+            >
+              <a
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://drive.google.com/file/d/1Xcn9GyAFnlVk7Yhxu1Xx77V2T2u4LLMj/view?usp=sharing"
```
Comment (ridhof) [inline app/routes/cfp._index.tsx:39]: it's not the correct url, please keep papercall io as the href

File: app/routes/cfp._index.tsx [hunk 2]
```diff
@@ -19,19 +19,34 @@ export const meta: V2_MetaFunction = () => {
...
-          </a>
-        </Button>
-      </article>
+      <div className="flex h-full w-full items-center bg-primary-100 md:max-h-full ">
+        <div className="m-auto flex w-full max-w-7xl flex-col items-center justify-between md:flex-row lg:h-[550px] lg:px-3">
+          <div className="px-4">
+            <p className="mb-6 text-4xl font-bold md:text-6xl">
+              Call For Proposals
+            </p>
+            <p className="text-xl text-primary md:text-4xl lg:text-2xl">
```
Comment (ridhof) [inline app/routes/cfp._index.tsx:28]: please use the same font styling with sponsorship page, because the font size doesn't look the same

**Review by ridhof — CHANGES_REQUESTED** [review]

**Comment by jeeehaan** [comment]
Mas @ridhof, I would like to request a review of my latest push. I just added responsiveness to the page, could you please take some time to review the updates?

**Review by ridhof — CHANGES_REQUESTED** [review]

**Comment by ridhof** [comment]
lgtm

responsiveness looks good, please solve request changes I mentioned above and then you can merge this PR

**Comment by jeeehaan** [comment]
Hi mas @ridhof, i've solved the request you mentioned above, but my merge button is currently blocked. Could you help me merge my PR?

**Review by ridhof — APPROVED** [review]
LGTM

**Pull request merged**